### PR TITLE
Adding new levels 11

### PR DESCRIPTION
**Pull request by shoc71**

### CODE REVIEW

**Comment by shoc71** [comment]
- 2 new levels

File: Keep On Going/assets/levels/005_Auto-complete.txt [hunk 1]
```diff
@@ -649,3 +649,46 @@ self.win_zones =  [
 ]
 
 =========
+
```
Comment (slysnivy) [inline Keep On Going/assets/levels/005_Auto-complete.txt:652]: This level doesn't have a level title (Which is crucial to how it's imported)

**Review by slysnivy — CHANGES_REQUESTED** [review]
I think the level title is missing for the 005_Auto-complete.

**Review by slysnivy — APPROVED** [review]
Things should be fine now.

**Pull request merged**